src/plugins/intel_cpu/tests/functional/single_layer_tests/interpolate.cpp

```diff
@@ -349,6 +329,244 @@ std::vector<std::map<std::string, std::string>> filterAdditionalConfig() {
     }
 }
 
+// 3D
+std::vector<CPUSpecificParams> filterCPUInfoForDevice3D() {
+    std::vector<CPUSpecificParams> resCPUParams;
+    if (InferenceEngine::with_cpu_x86_avx2()) {
+        resCPUParams.push_back(CPUSpecificParams{{ncw, x, x, x}, {ncw}, {"jit_avx2"}, "jit_avx2"});
+    } else {
+        resCPUParams.push_back(CPUSpecificParams{{ncw, x, x, x}, {ncw}, {"ref"}, "ref"});
+    }
+    return resCPUParams;
+}
+
+std::vector<std::map<std::string, std::string>> filterAdditionalConfig3D() {
+    return {
+        // default config as an stub
+        {{InferenceEngine::PluginConfigParams::KEY_PERF_COUNT, InferenceEngine::PluginConfigParams::NO}}
```

Comment:
Why exactly this option? Just leave empty config.

Reply:
I thought empty vector lead to zero test cases(0*x*y..) previously. I have tested that empty just not set config, it's still a stub.  Have updated, thanks Dmitry.